sunpy/map/tests/test_plotting.py

```diff
@@ -206,6 +206,13 @@ def test_draw_contours_aia(aia171_test_map):
     aia171_test_map.draw_contours(u.Quantity(np.arange(1, 100, 10), 'percent'))
 
 
+@figure_test
+def test_draw_contours_different_wcs(aia171_test_map):
+    rotated_map = aia171_test_map.rotate(30*u.deg)
```

Comment:
Just for speed reasons?
```suggestion
    rotated_map = aia171_test_map.rotate(30*u.deg, order=3)
```

Reply:
Done